service/frontend/operator_handler.go

```diff
@@ -280,18 +277,22 @@ func (h *OperatorHandlerImpl) addSearchAttributesSQL(
 	if nsName == "" {
 		return errNamespaceNotSet
 	}
-	ns, err := h.namespaceRegistry.GetNamespace(namespace.Name(nsName))
+	resp, err := client.DescribeNamespace(
```

Comment:
Is this to avoid cache? Why not to call `DescribeNamesapce` from namespace handler directly?

Reply:
Yes, it's to avoid cache. Namespace handler is not available in this context.

Reply:
It can be added, but ok, let's leave it this way.